From 4b809febe912ac084994b3297505c33a030d42f1 Mon Sep 17 00:00:00 2001
From: "Restyled.io" <commits@restyled.io>
Date: Mon, 3 Feb 2025 13:01:45 +0000
Subject: [PATCH] Restyled by whitespace

---
 src/app/globals.css | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/globals.css b/src/app/globals.css
index ba96303..749a8be 100644
--- a/src/app/globals.css
+++ b/src/app/globals.css
@@ -7,4 +7,4 @@
 .post-content {
     font-weight: default;
     font-size:default;
-}
\ No newline at end of file
+}
-- 
2.48.1
